README.md

```diff
@@ -43,27 +48,36 @@ The project uses test-drive agile development processes, where requirements are
 
 ### [](https://github.com/pushpinderpalsingh/SpaceDash#setup)Setup
 
-* Fork this project
-* Clone the forked project to your machine
-* In your project directory open Terminal and Run  `$ pod install`  and  `$ pod update`
-* Open the .xcworkspace file in Xcode to start working on the App
+1) Fork this project
+2) Clone the forked project to your machine
+3) In your project directory open Terminal and Run  `$ pod install`  and  `$ pod update`
+4) Open the `.xcworkspace` file in Xcode to start menu working on the App
+
+
+### [](https://github.com/pushpinderpalsingh/SpaceDash#join-the-development) Contribute with us!
+
+Before you start developing please read through [Contributing.md](https://github.com/pushpinderpalsingh/SpaceDash/blob/develop/Contributing.md) 
+
+#### Set up contributing environment
+
+Before you assist with the development, please set up the project on your local machine, run it, and go through the complete application 
+
+Click through buttons you can find and see where they take you. Explore the app in detail so that you are aware of core features.
 
+#### Issues
```

Comment:
I think contributing.md would be a better place for this

Reply:
Meaning I should move this entire section to contributing.md?

Reply:
Actually I just saw that there's an issues section that already exists! I'll combine them 🚀